master/internal/db/postgres_rp_workspace_bindings_intg_test.go

```diff
@@ -34,10 +35,53 @@ func TestListWorkspacesBindingRP(t *testing.T) {
 	return
 }
 
-func TestListRPsBoundToWorkspace(t *testing.T) {
+func TestListRPsAvailableToWorkspace(t *testing.T) {
 	// pretty straightforward
 	// don't list binding that are invalid
 	// return unbound pools too (we pull from config)
+	require.NoError(t, etc.SetRootPath(RootFromDB))
+	db := MustResolveTestPostgres(t)
+	MustMigrateTestPostgres(t, db, MigrationsFromDB)
+	ctx := context.Background()
+	user := RequireMockUser(t, db)
+
+	// A workspace has no bounded RP
+	// TODO: Write RP config to master config?
+	existingPools := []config.ResourcePoolConfig{
+		{PoolName: "poolName1"},
+		{PoolName: "poolName2"},
+		{PoolName: "poolName3"},
+	}
+
+	workspaceNames := []string{"workspace1"}
+	workspaceIDs, err := MockWorkspaces(workspaceNames, user.ID)
+	require.NoError(t, err)
+
+	rpWorkspaceBindings, _, err := ReadRPsBoundToWorkspace(ctx, workspaceIDs[0], 0, 500)
+	require.NoError(t, err)
+	boundRPs := []string{}
+	for _, rpWorkspaceBinding := range rpWorkspaceBindings {
+		boundRPs = append(boundRPs, rpWorkspaceBinding.PoolName)
+	}
+	expectedBoundRPs := []string{"poolName1", "poolName2", "poolName3"}
```

Comment:
this seems brittle - if we change the resource pool config we'll have to change this too. It might be bulkier, but I would prefer getting this list by looping over `existingPools`.

Reply:
Make sense will update.